data/human/hails.txt

```diff
@@ -2962,6 +2996,34 @@ phrase "hostile disabled"
 		"."
 		"!"
 
+# Shakespearean insults.
+phrase "disabled hostile"
+	word
+		"You piece of"
+		"Son of a"
+	word
+		" "
+	phrase
+		"shakespearean insults"
+	word
+		"!"
+		"."
+	word
+		" "
+	word
+		"Look what you did to my ship."
```

Comment:
```suggestion
		"Look what you did to my ship!"
```
Up to you. I think this seems like something that would be exclaimed rather than stated.

Reply:
Wanted to make three statements with each of the ending punctuation marks, simply as arbitrary as that. But if it helps to read better, there's no issue with what I intended